index.js

```diff
@@ -0,0 +1,71 @@
+const menuContainer = document.getElementById('toolbar');
+const iconImage = menuContainer.querySelector('img');
+let controller = false;
+menuContainer.addEventListener('click', () => {
+  if (!controller) {
+    document.querySelector('.desk').style.display = 'flex';
+    document.querySelector('.desk').style.background = '#fff';
+    document.querySelector('.desk').style.position = 'absolute';
+    document.querySelector('.desk').style.width = '100%';
+    document.querySelector('.desk').style.height = '100vh';
+    document.querySelector('.desk').style.top = '0px';
+    document.querySelector('.desk').style.position = 'fixed';
```

Comment:
- [ ] Kindly make sure to follow the best practices and avoid repeating the selector for the same property here, you can use a variable one time and add the style 
e.g 
```javascript
 const desk =  document.querySelector('.desk')
 desk.style.display = 'flex';
 ...
 ...
```
 or you can add the styles in the CSS file instead